website/js/app/small-screen-menu-behavior.js

```diff
@@ -0,0 +1,56 @@
+// When the window is small, #header-icon becomes a toggleable menu icon which will show/hide
+// #left-nav. (Which will then float on top of #content-container.) This file implements the
+// behavior of the menu icon.
+(function() {
```

Comment:
Ack. I've added a TODO with some notes, but will leave it as-is. Evidently you need to still let angular know about window resizes, since otherwise it wouldn't be aware that window.clientWidth is updated. (Or something.) A cursory search reveled this wasn't as simple as it sounds.